localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+echo ""
+echo "Install po2json..."
+echo ""
+npm install po2json
```

Comment:
We could move this step in CI config before [commit_language.sh](https://github.com/greenpeace/planet4-handbook/blob/develop/.circleci/config.yml#L310) & also the nodejs installation if applicable 🤔

Reply:
We could, though having it here has the advantage of having everything in once place, and I guess the installation should be fast anyway (on the order of seconds)?

Reply:
Yes, its a quick install.